src/copentime/errorStatus.cpp

```diff
@@ -24,9 +24,7 @@ OTIO_API const char *OpenTimeErrorStatus_outcome_to_string(
         OpenTimeErrorStatus *self, OpenTime_ErrorStatus_Outcome var1) {
     std::string returnStr = opentime::ErrorStatus::outcome_to_string(
             static_cast<opentime::ErrorStatus::Outcome>(var1));
-    char *charPtr = (char *) malloc((returnStr.size() + 1) * sizeof(char));
-    strcpy(charPtr, returnStr.c_str());
-    return charPtr;
+    return CppString_to_CString(returnStr);
```

Comment:
I think strdup is sufficient here TBH, plus _CRT_SECURE_NO_WARNINGS of course

Reply:
Good point, I had forgotten about ```strdup```.

Reply:
_CRT_NONSTDC_NO_WARNINGS ~~~ TIL ;)

Reply:
same note about API changes should be a separate PR